lib/api/controllers/securityController.js

```diff
@@ -315,7 +333,7 @@ class SecurityController {
     return this.kuzzle.repositories.user.load(request.input.resource._id)
       .then(user => {
         if (!user) {
-          return Promise.reject(new NotFoundError(`User with id ${request.input.resource._id} not found`));
+          return Bluebird.reject(new NotFoundError(`User with id ${request.input.resource._id} not found`));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 108 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)